servant/src/Servant/API/Delete.hs

```diff
@@ -12,9 +12,7 @@ import Data.Typeable ( Typeable )
 --
 -- >>>            -- DELETE /books/:isbn
 -- >>> type MyApi = "books" :> Capture "isbn" Text :> Delete
-data Delete (contentTypes :: [*]) a
-  deriving Typeable
-
+type Delete (contentTypes :: [*]) a = HttpMethod "DELETE" 200 contentTypes a
```

Comment:
Yes, I got quite confused starting out because the documentation says you can use `Delete` without arguments in a few places.